src/Servers/Kestrel/Transport.NamedPipes/src/Internal/NamedPipeConnectionListener.cs

```diff
@@ -78,6 +83,8 @@ private async Task StartAsync(NamedPipeServerStream nextStream)
 
                     await stream.WaitForConnectionAsync(_listeningToken);
 
+                    _log.LogInformation("Connection accepted on " + index);
```

Comment:
could this use a less allocatey overload (in particular when it turns out nobody wants the log message)?